recipes/scijava-jupyter-kernel/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -euo
+
+# Install Beakerx...
+# ...waiting for a Maven artifact.
+cd $RECIPE_DIR/
+
+git clone https://github.com/twosigma/beakerx.git
+cd beakerx/
+git checkout 585f07c5dfe7f9f0f97053d90ffc5a696d972382
```

Comment:
Ok I'll add a comment and open an issue once the feedstock repo is created. Thanks for your understanding.